index.html

```diff
@@ -53,6 +53,7 @@
                 <br>
             </div>
             <h3><a id="zen-video-title"></a></h3>
+            <div id="zen-video-description"></div>
```

Comment:
How do you feel about adding a button to toggle the description like we have for player? (ie: Show/Hide Description)